server/controllers/finance/reports/operating/index.js

```diff
@@ -77,9 +78,13 @@ function document(req, res, next) {
 
   fiscal.getDateRangeFromPeriods(periods).then(dateRange => {
```

Comment:
So, I never noticed this before, but it looks like we send the periods to the server, convert them into dates in this line, then later do:
```sql
      pt.period_id IN (
        SELECT id FROM period WHERE start_date>= ? AND end_date<= ?
      )
```
... which basically undoes the whole operation.  We probably could make this code cleaner by just using the period ids we have from the client directly.

This change could be another PR.